scss/_buttons.scss

```diff
@@ -92,11 +92,11 @@
 
   .counter {
     color: #60b044;
-    background-color: #fff;
+    background-color: $bg-white;
   }
 
   &:hover {
-    color: #fff;
+    color: $text-white;
     background-color: darken(#60b044, 5%);
     background-image: linear-gradient(darken(#8add6d, 5%), darken(#60b044, 5%));
```

Comment:
We'll need to be pretty careful if we do this since our buttons would end up looking almost identical to some other UI elements – e.g. labels and issue state badges (Open / Merged / Closed).